edx_sga/sga.py

```diff
@@ -263,53 +263,57 @@ def student_state(self):
             "upload_allowed": self.upload_allowed(),
         }
 
-    def staff_grading_data(self):
+    def staff_grading_data(self, student_user=None):
         """
         Return student assignment information for display on the
         grading screen.
         """
-        def get_student_data():
-            # pylint: disable=no-member
-            """
-            Returns a dict of student assignment information along with
-            annotated file name, student id and module id, this
-            information will be used on grading screen
-            """
-            # Submissions doesn't have API for this, just use model directly.
+        student_data = []
+
+        # Submissions doesn't have API for this, just use model directly.
+        if student_user is not None:
+            student_anonymous_id = anonymous_id_for_user(
+                user=student_user, course_id=self.course_id, save=False
+            )
+            students = SubmissionsStudent.objects.filter(
+                student_id=student_anonymous_id
```

Comment:
Is it relevant that this call isn't filtering by block_id and the previous one was?